ui/src/pages/version/components/forms/components/AutoscalingPolicyFormGroup.js

```diff
@@ -109,27 +109,33 @@ export const AutoscalingPolicyFormGroup = ({
           Autoscaling Policy determines the condition for increasing or
           decreasing number of replicas.
         </Fragment>
-      }>
+      }
+      fullWidth
```

Comment:
I've added all this `fullWidth` props everywhere in order to keep the width of all the components in the page consistent when the page is displayed on super large screens.